application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -69,7 +69,8 @@ public ReportCommand(Config config) {
 
         configModMailChannelPattern = config.getModMailChannelPattern();
 
-        configModGroupPattern = config.getHeavyModerationRolePattern();
+        configModGroupPattern =
+                Pattern.compile(config.getSoftModerationRolePattern()).asMatchPredicate();
```

Comment:
please ping the `Heavy` not the `Soft` mod role. The channel can only be seen by former. Thanks.